README.md

```diff
@@ -1,64 +1,68 @@
 # Team Project Versionning
 
 ## Summary
+
 - [Topic](#our-topic)
 - [Git flow](#git-flow)
-    - [Branches](#our-branches)
-    - [How it works](#how-git-flow-works)
-- [GITHUB workflow](#our-github-worflow)
-    - [Issues](#create-an-issue)
-    - [Board](#our-project-board)
-    - [PR](#our-pr)
--[Checks](#check-after-push)
-    - [html5validator](#html5validator)
+  - [Branches](#our-branches)
+  - [How it works](#how-git-flow-works)
+- [GITHUB workflow](#our-github-worflow) - [Issues](#create-an-issue) - [Board](#our-project-board) - [PR](#our-pr) -[Checks](#check-after-push) - [html5validator](#html5validator)
```

Comment:
Why all of this different point are on the same line now ?? 

The view is not correct now